setup.py

```diff
@@ -10,16 +9,22 @@
 
 setuptools.setup(
     name="honeybee-core",
-    use_scm_version = True,
+    use_scm_version=True,
     setup_requires=['setuptools_scm'],
     author="Ladybug Tools",
     author_email="info@ladybug.tools",
-    description="Honeybee is a Python library to create, run and visualize the results of environmental simulation. See plugins (e.g. honeybee_radiance) for specific simulation type.",
+    description="Honeybee is a Python library to create, run and visualize the results of environmental simulation. See extensions (e.g. honeybee_radiance) for specific simulation type.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/ladybug-tools/honeybee-core",
     packages=setuptools.find_packages(exclude=["tests"]),
     install_requires=requirements,
+    extra_requires={
+        'cli': ['click>=5.1']
```

Comment:
Is this the section that governs the `pip install honeybee-core[cli]` behaviour?

Reply:
yes. the only difference is by adding cli it will install `click`. If you have click installed then `pip install honeybee-core[cli]` and `pip install honeybee-core` will be the same.

I think it will be really useful for us to set the main libraries like `lbt-honeybee` where we can use this tag to install or not install dependencies like `honeybee-radiance`, `honeybee-energy` and so on: `pip install lbt-honeybee[radiance]` and one can always do `pip install lbt-honeybee[all]` to get everything installed.

Reply:
That is a solid shout. It does mean we should have a bit of a think about dependency management and all of that (as in: plugin architecture is flexible so long as all things are not inter-dependent)